pybess/bess.py

```diff
@@ -70,20 +73,22 @@ def _import_modules(name, subdir):
     error here."""
     def protobufs(subdir):
         "Yield modules (subdir=None) or port drivers (subdir='ports')."
-        self_path = os.path.dirname(os.path.relpath(__file__))
-        for directory in ('builtin_pb', 'plugin_pb'):
-            if subdir:
-                dirpath = os.path.join(self_path, directory, subdir)
-                prefix = '..{}.{}'.format(directory, subdir)
-            else:
-                dirpath = os.path.join(self_path, directory)
-                prefix = '..{}'.format(directory)
-            for filename in os.listdir(dirpath):
-                if not filename.endswith('_msg_pb2.py'):
-                    continue
-                import_name = '{}.{}'.format(prefix, filename[:-3])
-                if import_name != '..builtin_pb.bess_msg_pb2':
-                    yield import_name
+        for path in sys.path:
+            if not os.path.exists(os.path.join(path, 'builtin_pb')):
+                continue
+            for directory in ('builtin_pb', 'plugin_pb'):
+                if subdir:
+                    dirpath = os.path.join(path, directory, subdir)
+                    prefix = '{}.{}'.format(directory, subdir)
+                else:
+                    dirpath = os.path.join(path, directory)
+                    prefix = '{}'.format(directory)
```

Comment:
nit: This can be
```
prefix = directory
```

Reply:
This one i'm going to leave as-is because it better matches the form of the one above it.